packages/eas-cli/scripts/verify-graphql.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+function cleanup()
+{
+  echo 'Cleaning up...'
+  rm -f graphql.schema.json.bak src/graphql/generated.ts.bak
+}
+
+# Fail if anything errors
+set -eox pipefail
+# If this script exits, trap it first and clean up
+trap cleanup EXIT
+
+cp graphql.schema.json graphql.schema.json.bak
+cp src/graphql/generated.ts src/graphql/generated.ts.bak
+
+yarn generate-graphql-code
+if cmp -s graphql.schema.json graphql.schema.json.bak; then
+    echo "GraphQL schema is up-to-date"
+    if cmp -s src/graphql/generated.ts src/graphql/generated.ts.bak; then
+        echo "GraphQL generated code is up-to-date"
+        exit 0
+    else
+        echo "GraphQL code has changed but has not been regenerated. Run `yarn generate-graphql-code` and commit the changes.""
+        exit 1
+    fi
+else
+    echo "GraphQL schema has changed on the server. Run `yarn generate-graphql-code` and commit the changes."
+    exit 1
```

Comment:
```suggestion
    echo "GraphQL schema has changed on the server. Running `yarn tsc` to check for breaking changes..."
    yarn tsc
    # If there are no errors, then the schema is backwards compatible
    exit 0
```

Reply:
@wschurman Does this look correct for handling server side GraphQL changes?

Reply:
I think so. The difference is that I don't think we need to compare the `generated.ts` since there's no way to just get a new schema but not regenerate the ts.